install.sh

```diff
@@ -77,7 +78,7 @@ echo "================= Adding gcloud ============"
 CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)"
 echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee /etc/apt/sources.list.d/google-cloud-sdk.list
 curl -sS https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
-sudo apt-get update && sudo apt-get install google-cloud-sdk=157.0.0-0
+sudo apt-get update && sudo apt-get install google-cloud-sdk=160.0.0-0
```

Comment:
157.0.0.0-0 isn't available any longer. https://cloud.google.com/sdk/docs/release-notes lists one or two breaking changes, but its unlikely to affect our users.